src/SOS/Strike/exts.cpp

```diff
@@ -176,32 +238,19 @@ ArchQuery(void)
     g_targetMachine = GetTargetMachine(processorType);
     if (g_targetMachine == NULL)
     {
-        const char* architecture = "";
-        switch (processorType)
+        const char* architecture = GetProcessorName(processorType);
+        const char* message = "";
+#if defined(SOS_TARGET_AMD64) || defined(SOS_TARGET_ARM64)
+        if (processorType == IMAGE_FILE_MACHINE_ARM64EC)
+        {
+            message = "arm64ec targets require a x64 SOS and debugger.";
```

Comment:
Arm64EC works too.

Reply:
Not sure what you are saying? That I should capitalized it like that?

Reply:
That an ARM64EC debugger works (anything capable of running x64 SOS)